modules/ding_innskraning/test/SAML_response.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why is this included? Consinder explaining this somewhere - perhaps in a comment at the beginning of this file?